include/geode/io/mesh/private/vtk_input.h

```diff
@@ -222,28 +229,21 @@ namespace geode
                         attribute->set_value( i + offset, values[i] );
                     }
                 }
+                else if( nb_components == 2 )
+                {
+                    create_attribute< std::array< T, 2 >, T >(
+                        manager, {}, values, nb_components, name, offset );
+                }
+                else if( nb_components == 3 )
+                {
+                    create_attribute< std::array< T, 3 >, T >(
+                        manager, {}, values, nb_components, name, offset );
+                }
                 else
```

Comment:
https://github.com/Geode-solutions/OpenGeode/blob/e394d0b889f580431d1332401db6f2cd6a1177d3/include/geode/basic/detail/bitsery_archive.h#L47

here we have register also array of 4 T. Maybe add another else if to be homogeneous

Reply:
Maybe. Is 4D really something to output in other file format ?